docs/posts/ibis-version-v8-release/index.qmd

```diff
@@ -0,0 +1,238 @@
+---
+title: "Ibis v8: streaming and more!"
+author: "Ibis team"
+date: "2024-02-05"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis v8 marks the first release of data streaming backends in Ibis! We're
+excited to begin bridging the gap between batch and streaming systems, starting
+with [Apache Flink](https://flink.apache.org/) and
+[RisingWave](https://risingwave.com) backends in this release.
+
+We've also added a new batch backend with [Exasol](https://exasol.com), bringing
+the total number of backends Ibis supports to 20+.
+
+### What is data streaming?
+
+Data streaming is a method of processing data continuously as it arrives rather
+than in batches at points in time. This is useful for applications that require
+low latency and increasingly common in modern data applications.
+
+Apache Flink is the most popular open-source stream processing framework, with
+numerous cloud options. RisingWave is an open-source Postgres-compatible
+streaming database with a cloud offering that is gaining popularity and
+simplifies the streaming experience.
+
+Ibis now supports both and going forward can add more streaming backends to
+unify the Python user experience across batch and streaming systems.
```

Comment:
maybe a qualifier here -- in some respects we've already unified this

Reply:
how if we didn't have a streaming backend before?

Reply:
maybe it's the unclear antecedent.

What is allowing the unification?  That we "now support both" or that "going forward [we] can add more to unify..."